### DISCUSSION TITLE
Change cert control from Self cert to Strimzi cert without kafka restart

### DISCUSSION THREAD

**Discussion by sancdevops** [strimzi/strimzi-kafka-operator#9166] [Q&A]
Hi All , 
Need your help regarding a scenerio . strimzi version 0.28
1. We installed a kafkacluster and it generated a certificate for Cluster and client. 
 generateCertificateAuthority=true
2. For some reason the certs generated are kept and kafka cluster is set as 
generateCertificateAuthority=false(This removed the certs control for kafka and cluster was brought downfor that )
3. Now we wanna again set 
generateCertificateAuthority=true (As cert renewals aree approching ) . But we cant make the cluster down. 

So how can i set the flag without restarting the cluster .

**Comment by sancdevops** [comment]
any update ?? @scholzj , @schmichri

**Reply by scholzj** [reply]
0.28.0 is quite old and I do not remember anymore what changed, so I cannot really help. Sorry.

**Comment by sanchayana2007** [comment]
But can you point the process/docs on how to make strimzi recognize my certAuth changes without reinstalling kafka .
